package.json

```diff
@@ -11,6 +11,8 @@
     "@aws-sdk/client-dynamodb": "^3.899.0",
     "@aws-sdk/client-sqs": "^3.899.0",
     "@aws-sdk/client-ssm": "^3.777.0",
+    "@aws-sdk/client-s3": "^3.899.0",
```

Comment:
Hi @peterfajemisincabinetoffice & @danacotoran - Could we create a ticket, in the SFAD epic, to create an opinionated AWS KMS client like we have already DynamoDB and SQS please - and then we can swap it in later - thanks